store/src/java/com/zimbra/cs/session/SoapSession.java

```diff
@@ -479,6 +480,7 @@ void clearMailboxChanges() {
     private boolean isOffline = false;
     private final SoapProtocol responseProtocol;
     private String curWaitSetID;
+    protected AuthToken authToken;
```

Comment:
instead of storing the auth token object, better to store the encoded string: authToken.getEncoded()